### PR TITLE
plugin command -create,inspect,list,remove-

### PR DESCRIPTION
**Pull request by DomTheDon12**
Plugin command support

### CODE REVIEW

**Comment by DomTheDon12** [comment]
Hi!
I'm a university student, and this was my task to do in one of our courses. Can you help me what's the problem?

**Comment by DomTheDon12** [comment]
Anyone?

**Comment by trajano** [comment]
@DomTheDon12 I think something is wrong with the Travis build itself.    https://github.com/docker-java/docker-java/pull/1366 just adds enums and the build fails.

**Comment by DomTheDon12** [comment]
I'm new at docker-java. How can I try my code (plugin commands) if it's working or not? Can you help me with that?

**Comment by trajano** [comment]
You may want to rebase and push again, I think they fixed the builds

**Comment by bsideup** [comment]
Hi @DomTheDon12,

Thanks for working on this! Could you please add tests?

File: docker-java/src/test/java/com/github/dockerjava/junit/DockerCmdExecFactoryDelegate.java [hunk 1]
```diff
@@ -24,4 +28,15 @@ public void init(DockerClientConfig dockerClientConfig) {
             ((DockerClientConfigAware) delegate).init(dockerClientConfig);
         }
     }
+    @Override
+    public ListPluginsCmd.Exec listPluginsCmdExec() { return delegate.listPluginsCmdExec(); }
```
Comment (bsideup) [inline docker-java/src/test/java/com/github/dockerjava/junit/DockerCmdExecFactoryDelegate.java:32]: Please fix the code style here and in other places:
```suggestion
    public ListPluginsCmd.Exec listPluginsCmdExec() {
        return delegate.listPluginsCmdExec();
    }
```

**Pull request closed without merging**